### DISCUSSION TITLE
How can i save array of Bitmap object to animated Gif

### DISCUSSION THREAD

**Discussion by workgroupengineering** [AvaloniaUI/Avalonia#6084] [Q&A] [answered]
How can i save array of Bitmap object to animated Gif

**Comment by sn4k3** [comment, marked as answer]
I use this in my project: https://github.com/mrousavy/AnimatedGif
Works great!

To render a GIF on your avalonia app: https://github.com/AvaloniaUI/Avalonia.GIF
